runelite-client/src/main/java/net/runelite/client/plugins/wintertodt/WintertodtPlugin.java

```diff
@@ -205,6 +210,27 @@ private void checkActionTimeout()
 		}
 	}
 
+	@Subscribe
+	public void onGameObjectSpawned(GameObjectSpawned event)
+	{
+		if (!config.notifyFallingSnow() || !isInWintertodt())
+		{
+			return;
+		}
+
+		Player player = client.getLocalPlayer();
+		GameObject gameObject = event.getGameObject();
+
+		if (WINDERTODT_FALLING_SNOW == gameObject.getId())
+		{
+			if (gameObject.getLocalLocation().equals(player.getLocalLocation()))
```

Comment:
Isnt the falling snow AOE? e.g, will this actually work?

Reply:
I tested it and it worked.